app/inject/I18nResolverProvider.java

```diff
@@ -32,7 +32,7 @@ public I18nResolverProvider(final Configuration configuration, final ProjectCont
 
     @Override
     public I18nResolver get() {
-        final List<Locale> locales = projectContext.languages();
+        final List<Locale> locales = projectContext.locales();
```

Comment:
This makes it harder to understand. So what is locales?
So if you have language "de" and have country "DE" you still need locale "de-DE" for formatting prices right. But you want to search for language "de".